crates/env/src/engine/on_chain/ext.rs

```diff
@@ -589,6 +593,19 @@ pub fn set_rent_allowance(value: &[u8]) {
     unsafe { sys::seal_set_rent_allowance(Ptr32::from_slice(value), value.len() as u32) }
 }
 
+pub fn rent_params(output: &mut &mut [u8]) {
```

Comment:
Need to be put in a
```rust
#[link(wasm_import_module = "__unstable__")]
extern "C" { }
```
block.

Reply:
why not just `"unstable"` btw?

Reply:
I use it to indicate that it has some special semantic and is not only a module name.